src/Snppts/Snippets/FoodApp.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using Snppts.Authors;
+using Snppts.Infrastructure;
+
+namespace Snppts.Snippets
+{
+    public class FoodApp : IAmASnippet
+    {
+        public string Slug => "foodapp";
+        public string Title => "foodapp";
+        public string GithubRepoName => "monetelli/XF_FoodApp";
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => true;
```

Comment:
Just double-checking here. You say there is a UWP app, I see it in source but nowhere on screenshots? The UWP app is functioning as well?

Reply:
That's right, but since it's in the preview phase of Shell and updating Xamarin.Forms 4.3 ... it has had some drawbacks.

Reply:
Do you think that if I take it as a false UWP and then, when it is stable, I make it true?

Reply:
No, it's fine!  Was just checking that people wouldn't be disappointed whenever they navigated to your website and didn't find any UWP content at all :)